config/peerpods/podvm/lib.sh

```diff
@@ -198,8 +198,12 @@ function prepare_source_code() {
 
     if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
         sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/; s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
-            "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
-        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+            "${podvm_dir}"/files/etc/kata-opa/default-policy.rego >"${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
+        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+
+        # Remove the root disk mounts from the podvm
+        rm -f "${podvm_dir}"/files/etc/systemd/system/{run-image,run-kata\\x2dcontainers}.mount
+        rm -f "${podvm_dir}"/files/etc/systemd/system/multi-user.target.wants/{run-image,run-kata\\x2dcontainers}.mount
```

Comment:
Can there be other mount files from some custom podvm layout ? Should'nt they be removed all the same, i.e. `find "${podvm_dir}"/files/etc/systemd/system --name *.mount --delete` ?

Reply:
Hmm the image is generated with files that is coming from known sources and contents are known. So finding and deleting may not be needed. Wdyt ?

Reply:
If this list of known sources is authoritative, then it is fine. Maybe add a comment with a link to the CAA repo where people can check the list ?

Reply:
Just curious, why the `\x2d` and not `-`? It looks weird.

Reply:
Since the answer was given to me on Slack, here it is: https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#String%20Escaping%20for%20Inclusion%20in%20Unit%20Names

So `run-kata\x2dcontainers` is the `systemd` unit spelling for `run/kata-containers`.

Reply:
Adding the systemd document link here for reference - https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#String%20Escaping%20for%20Inclusion%20in%20Unit%20Names